gcp/appengine/frontend_handlers.py

```diff
@@ -515,6 +517,16 @@ def group_versions(versions):
   return groups
 
 
+def sort_version(versions):
+  """Sorts a list of version numbers in natural order."""
+  try:
+    return sorted(versions, key=Version)
```

Comment:
This needs to be ecosystem specific, rather than always relying on `packging.version.Version`, which is Python's implementation.

Every ecosystem (npm, PyPI, NuGet etc) tend to have its own version ordering rules, which we implement here: https://github.com/google/osv.dev/blob/6551ae4c1877447e01896ed0fd32f797adf4d6fd/osv/ecosystems/_ecosystems.py#L88

Sample usage: https://github.com/google/osv.dev/blob/6551ae4c1877447e01896ed0fd32f797adf4d6fd/osv/models.py#L791

Reply:
Thanks!